apps/desktop/src/renderer/screens/main/components/WorkspaceView/WorkspaceInitializingView/KeypadLoader/KeypadLoader.css

```
@@ -0,0 +1,147 @@
+.keypad-loader {
+	--travel: 26;
+	position: relative;
```

Stylelint (17.7.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)